code/modules/mining/lavaland/loot/tendril_loot.dm

```diff
@@ -375,11 +375,12 @@
 
 /obj/item/projectile/hook/on_hit(atom/target)
 	. = ..()
-	if(isliving(target))
-		var/mob/living/L = target
-		if(!L.anchored)
-			L.visible_message("<span class='danger'>[L] is snagged by [firer]'s hook!</span>")
-			L.forceMove(get_turf(firer))
+	if(istype(target, /atom/movable))
```

Comment:
meat hook now actually moves your target towards you, rather than using forcemove